experimental/rank1_bnns/cifar.py

```diff
@@ -23,8 +23,10 @@
 from absl import logging
 
 import edward2 as ed
+import numpy as np
 from baselines.cifar import utils  # local file import
 from experimental.rank1_bnns import cifar_model  # local file import
+from experimental.rank1_bnns.refining import sample_rank1_auxiliaries
```

Comment:
It is preferable to import modules vs. functions directly, i.e., `from experimental.rank1_bnns import refining`.

Reply:
Changed imports in both files!